string robot_name
int32 camera_pixel
int32 width
int32 height